_topic_maps/_topic_map.yml

```diff
@@ -539,6 +539,8 @@ Topics:
   Distros: openshift-enterprise
 - Name: Updating hardware on nodes running on vSphere
   File: updating-hardware-on-nodes-running-on-vsphere
+- Name: Updating a cluster that includes the SRO
```

Comment:
maybe a better term is upgrading, instead of updating?

Reply:
guideline
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#update says "update" is the preferred term.